soc/arm/microchip_mec/common/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_include_directories_ifdef(CONFIG_SOC_SERIES_MEC172X .)
+zephyr_library_sources_ifdef(CONFIG_SOC_SERIES_MEC172X
+  soc_i2c.c
```

Comment:
minor nit.

Line is not that long so you must wrap it.

Reply:
@tejlmand maybe https://github.com/cheshirekow/cmake_format :-)?

Reply:
Thanks, worth taking a look at.